### ISSUE TITLE
Make line-break after breaking change prefix optional

### ISSUE THREAD

**Issue by SukumarMehta** [leonardoanalista/cz-customizable#88]
https://github.com/leonardoanalista/cz-customizable/blob/eee6212ca0a2dc764d1bc7ee4dfcff8ac6a1d0a9/buildCommit.js#L93

Looks like the referenced line mandates a \n between the prefix and the body of the breaking change.
The line break actually breaks some common semver parsing tools in gradle.  Is it possible to make this optionally a space rather than a line-break?

**Comment by leonardoanalista** [comment]
👀 ...

**Comment by leonardoanalista** [comment]
yes, totally possible. That's why this lib is called "customizable".

So this is how it is today (sample from this lib git log):
```
    feat: add inquirer as dependency

    also upgraded commitizen to the latest.

    BREAKING CHANGE:
    please reinstall the latest 2.9.5 commitizen globally via npm -i commitizen. Deprecated
    configuration option has been removed.

    ISSUES CLOSED: #34
```

Then we create a new config option. Let's say we call it: `breakingSectionBreakLineQuantity, {Integer} default 2`.

Then if you set to `0`, text will be in the same line of the body. If you set to `1`, text would be in the next line. Default would be as per sample code above. 

Next steps:
* What do you think? 
* Let's consider other sections to have the same capability in future. Name should be consistent. So what name?
* default value shouldn't break existing users or change the current behaviour. Reason: existing pattern suits most users that follow Angular commit convention.
* anything else?

**Comment by SukumarMehta** [comment]
Thanks for the quick response @leonardoanalista!

Just to clarify, using your example, I'm taking about the line break between "BREAKING CHANGE:" and "please reinstall..."

you mentioned a default of 2, if you wanted to avoid breaking current users this should be 1 (not 2) - correct?

**Comment by leonardoanalista** [comment]
you are correct. Default could be `1` to prevent breaking.

**Comment by shinobi5** [comment]
Is this update as simple as putting a ternary inside that template literal (based on the config option value) to decide whether to apply that `\n` or not?

Or is there another way you'd prefer this done @leonardoanalista?

I'm happy to put a PR up for this as I'd like this options as well

**Comment by leonardoanalista** [comment]
I think the default option should be as it is today and don’t introduce a braking change. The new opinion would be an opt-in. That’s what I think. I’m open to hear alternatives too

**Comment by shinobi5** [comment]
I agree.

So default: `breakingSectionBreakLineQuantity: 1` then the user can set value for the amount of line breaks they want (or set it to 0 if no line breaks required). 

Personally I don't see the utility in adding more than a single line break but I but can see the flexibility of giving the option to set the amount of line breaks if someone wanted to do this.

**Comment by leonardoanalista** [comment]
That's a good point. I can't see anyone adding 3 lines between blocks. I can see 0 or 1 being useful. Maybe that's why you were thinking of a `ternary` condition. I don't mind if this option is binary or not. I'll would approve a PR for either implementation. I guess whatever is simpler, which imo is `ternary`, therefore binary option - on/off.

**Comment by shinobi5** [comment]
Cool, sounds good. 

All good if I go ahead and create a PR for this? (probably in the next couple days...)

**Comment by shinobi5** [comment]
Hey @leonardoanalista.

Just forked the repo (so I can get a PR up for this issue) and kicked off the tests just to make sure everything is running OK. 

But I'm getting quite a few Type and Syntax errors ...

Using node `v13.14.0`

Steps:
1. `npm install`
2. `npm run test`

Do you get the same?
